api/machinemanager/machinemanager.go

```diff
@@ -36,3 +37,34 @@ func (client *Client) AddMachines(machineParams []params.AddMachineParams) ([]pa
 	}
 	return results.Machines, err
 }
+
+// DestroyMachines removes a given set of machines.
+func (client *Client) DestroyMachines(machines ...string) ([]params.DestroyMachineResult, error) {
```

Comment:
Can you add a todo to the CLI to use this new api instead of the one on the client facade?
And a deprecation todo on the client facade.